src/main/java/anticope/rejects/modules/AutoEnchant.java

```diff
@@ -0,0 +1,130 @@
+package anticope.rejects.modules;
+
+import anticope.rejects.MeteorRejectsAddon;
+
+import meteordevelopment.meteorclient.events.game.OpenScreenEvent;
+import meteordevelopment.meteorclient.settings.*;
+import meteordevelopment.meteorclient.utils.network.MeteorExecutor;
+import meteordevelopment.meteorclient.utils.player.FindItemResult;
+import meteordevelopment.meteorclient.utils.player.InvUtils;
+import meteordevelopment.orbit.EventHandler;
+import net.minecraft.enchantment.EnchantmentHelper;
+import net.minecraft.item.Item;
+import net.minecraft.item.Items;
+import net.minecraft.screen.EnchantmentScreenHandler;
+import net.minecraft.screen.ScreenHandler;
+
+import java.util.List;
+import java.util.Objects;
+
+public class AutoEnchant extends meteordevelopment.meteorclient.systems.modules.Module {
+
+    public final SettingGroup sgGeneral = settings.getDefaultGroup();
+
+    private final Setting<Integer> delay = sgGeneral.add(new IntSetting.Builder()
+            .name("delay")
+            .description("The tick delay between enchanting items.")
```

Comment:
the delay is in milliseconds

Reply:
Yes, although the description here indicates that the tick is the unit, the actual delay is not the tick unit, but it does not affect the normal operation in the game. Perhaps we can ignore it

Reply:
i would replace "tick" with "millisecond" in the description, because the current description is misleading